### ISSUE TITLE
Import/Export relational Field Names not clear enough

### ISSUE THREAD

**Issue by erondpowell** [directus/directus#12735]
### Describe the Bug

A minor stylistic issue, but I am purposing we add some way to identify the collection that a related Field comes from.

In the video, I add `translations.body` onto the `blog_posts` collection, then in the app interface (as well as the CSV) it is not made clear this is from the translation field, it simply says "body".

https://user-images.githubusercontent.com/62306630/163214773-59055040-f675-4647-a394-eb7329f46a54.mov

Another note: which may require a separate issue. Once the CSV is downloaded,  `body` is merged into `translations` and becomes a list of JSON objects, with each key saying "body" with the translation as the value, but not indicating _which translation_ it is. Perhaps we could change this so that we know which translation each "body" belongs to.

<img width="1698" alt="Screen Shot 2022-04-13 at 10 25 05 AM" src="https://user-images.githubusercontent.com/62306630/163216587-4464188d-bf0f-4fec-8271-ece796664c7d.png">

Also, I'm just now messing with this and potential _user error_ is lingering in the back of my mind.

### To Reproduce

configure an O2M or M2M and add multiple related Fields for export.

### Errors Shown

none.

### What version of Directus are you using?

9.8.0

### What version of Node.js are you using?

cloud

### What database are you using?

cloud

### What browser are you using?

chrome

### How are you deploying Directus?

Cloud
